src/tests/hitch_test.sh

```diff
@@ -182,15 +182,25 @@ hitch_pid() {
 }
 
 #-
-# Usage: hitch_hosts
+# Usage: hitch_worker_pids
 #
-# Print a list of hosts for the daemon started with `start_hitch`, usually in
+# Print the PIDs of the children of the daemon started with `start_hitch`
+
+hitch_worker_pids() {
+	ps --ppid "$(hitch_pid)" | grep -Eo  "^[ ]*[0-9]+"
```

Comment:
OpenBSD's [ps(1)](https://man.openbsd.org/ps) doesn't know about `--ppid`, but [pgrep(1)](https://man.openbsd.org/pgrep) does via the short option `-P`, which behaves the same as FreeBSD's pgrep(1) and Debian's [pgrep(1)](https://manpages.debian.org/pgrep).

As simple

     pgrep -P "$(hitch_pid)"

is therefore portable and much simpler as it already lists child PIDs one per line.